src/services/saneKeyboardEvents.util.js

```diff
@@ -109,7 +109,7 @@ var saneKeyboardEvents = (function() {
       clearTimeout(timeoutId);
       timeoutId = setTimeout(checker);
     }
-    target.bind('keydown keypress input keyup focusout paste', function() { checkTextarea(); });
+    target.bind('keydown keypress input keyup focusout paste', function(e) { checkTextarea(e); });
```

Comment:
NBD, but `function (e) { checkTextArea(e); }` could just be `checkTextArea`, right? Unless the are extra arguments you're trying to suppress.